spec/data/example_specs.rb

```diff
@@ -1,54 +1,64 @@
 require 'spec_helper'
-include Apivore::RspecBuilder
 
 context "API testing scenarios" do
-  before do
-    apivore_setup '/services/{id}.json', 'get', '200' do
-      {'id' => 1}
-    end
+  include Apivore::RspecMatchers
+  include Apivore::RspecHelpers
 
-    apivore_setup '/services.json', 'post', '204' do
-      {"_data" => {'name' => 'hello world'}}
+  describe "undocumented path in test", :type => :request do
```

Comment:
This test is meant to confirm that a correctly documented path that is not yet implemented in the mock API will show up in the output.

does `describe "unimplemented path in API", :type => :request do` convey that?

There will need to be wording changes in spec/integration_spec.rb too

The current failure output for the test is:
`Swagger doc: /02_unimplemented_path.json does not have a documented path for /path_does_not_exist`

which is not what the test is trying to do -- it should be testing for the documented path `/not_implemented.json`, which will fail when that request is made to the mock API